docs/source/basic_usage.rst

```diff
@@ -238,3 +238,10 @@ and a pre-configured issue collector.
    ``"Bearer ${TYPHOS_JIRA_TOKEN}"``.
 4. ``TYPHOS_JIRA_EMAIL_SUFFIX`` (str): the default e-mail suffix to put on
    usernames, such as ``"@example.com"``.
+
+
+Launching the Examples
+======================
+There are example screens in the ``typhos.examples`` submodule. After
+installing ``typhos``, you can launch them using e.g.
+``python -m typhos.examples.positioner``.
```

Comment:
Do you think I should list them here? I think I should, I was just a little bit lazy when writing the docs since updating the docs when more are added is pain.

Reply:
I think it'd be worthwhile to. Copy/paste is perfect for lazy users like me.